content/blog/2022-01-18-reproducibility-and-support-bjoern/index.md

```diff
@@ -0,0 +1,24 @@
+---
+site: freiburg
+tags: [comment]
+title: Galaxy as a sharing infrastructure and how this enables efficient user support
+authors: "Bjoern Gruening, Cristóbal Gallardo, Ross Lazarus"
```

Comment:
What about Jen? I don't mind adding many authors that do support this blog.

Reply:
@bgruening  - sure but who is Jen ? :) Need a name to fix it.

Reply:
oh - Jen! Sure. I'll add her.